2022-12-21 22:08:10,464 fflow.py initialize [line:259] INFO Using Logger in `utils.logger.basic_logger`
2022-12-21 22:08:10,465 fflow.py initialize [line:260] INFO Initializing fedtask: pill_classification
2022-12-21 22:08:11,410 fflow.py initialize [line:276] INFO Using model `resnet18` in `benchmark.pill_classification.model.resnet18` as the globally shared model.
2022-12-21 22:08:11,415 fflow.py initialize [line:304] INFO Using sampler `threshold_sampler` in `sampler.threshold_sampler` as the globally shared model.
Using cache found in /home/ace15032jo/.cache/torch/hub/pytorch_vision_v0.10.0
2022-12-21 22:08:17,677 fflow.py initialize [line:324] INFO Initializing Clients: 10 clients of `algorithm.fedprox.Client` being created.
2022-12-21 22:08:17,678 fflow.py initialize [line:345] INFO Initializing Server: 1 server of `algorithm.fedprox.Server` being created.
Traceback (most recent call last):
  File "/home/ace15032jo/Projects/SampleSelection_easyFL/main.py", line 33, in <module>
    main()
  File "/home/ace15032jo/Projects/SampleSelection_easyFL/main.py", line 21, in main
    server = flw.initialize(option)
  File "/home/ace15032jo/Projects/SampleSelection_easyFL/utils/fflow.py", line 350, in initialize
    server = getattr(server_module, "Server")(
TypeError: Server.__init__() got an unexpected keyword argument 'device'